examples/device/ep/nixl_ep/buffer.py

```
@@ -853,23 +853,26 @@ def all_gather_object(obj):
         else:
             self.runtime.connect_ranks(remote_ranks, None, ipc_handles)
 
-    def connect_ranks(self, remote_ranks: List[int]) -> None:
+    def connect_ranks(self, remote_ranks: List[int], activate: bool = True) -> None:
```

Ruff (0.15.11)

[warning] Boolean-typed positional argument in function definition

(FBT001)